maraboupy/MarabouNetworkNNet.py

```diff
@@ -218,11 +447,204 @@ def nodeTo_f(self, layer, node):
             return node
         else:
             offset = self.layerSizes[0]
-            offset += sum([x*2 for x in self.layerSizes[1:layer]])
+            offset += sum([x * 2 for x in self.layerSizes[1:layer]])
             offset += self.layerSizes[layer]
 
             return offset + node
 
+    def getVariable(self, layer, node, f=True):
```

Comment:
Docstrings needs to have an empty line before "Args:" and before "Returns:". Also, "Return:" should be changed to "Returns:" Also, the first line should be a quick description, followed by an empty line, and then a longer description can be written in a separate section. So I'd recommend an empty line between """Returns variable ... and If f=True lines.

Also, if these functions are private and only useful for creating the MarabouNetworkNNet class, the :meta private: line should be added to the end of the docstring.

It would also be great to add some sort of quick description of the arguments, using the format:
variable (type): Description

These comments applies to the other functions that have been created.